android/samples/test-app/src/main/java/com/liferay/mobile/screens/testapp/customviews/UserPortraitInitialsView.java

```diff
@@ -38,6 +38,8 @@ protected void onFinishInflate() {
 	public void showPlaceholder(User user) {
 		portraitImage.setImageBitmap(null);
 
+		textView.setBackground(getResources().getDrawable(R.drawable.rounded_corner, null));
```

Comment:
Use ContextCompat instead of getResources, passing the theme as null is a code smell